general/README.md

```diff
@@ -4,4 +4,104 @@ Guidelines and best practices used by Eventbrite to provide consistency and prev
 
 ## Table of Contents
 
-Coming soon...
+0. [Conditionals](#conditionals)
+
+
+## Conditionals
+
+### Complex conditional expressions
+
+Complex conditional expressions within conditional statements can make understanding code challenging because you need to fully understand the expression in order to be able to process the overall code flow. In addition, in code reviews (or revisiting old code), it's difficult to know if the logic is correct without knowing what state the conditional expression is trying to represent.
+
+To make things easier, store complex conditional expressions in state variables:
+
+```js
+// good
+var shouldContinue = !options || (options.continue && options.hasWon);
+
+if (shouldContinue) {
+
+}
+
+// bad
+if (!options || (options.continue && options.hasWon)) {
+
+}
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+### Negative conditional expressions
+
+Positive conditional expressions are generally easier to follow than negative ones. Try to flip the logic so that the positive case is considered first:
+
+```js
+// good
+if (something && somethingElse) {
+    /* do stuff*/
+} else {
+    /* do other stuff */
+}
+
+// bad (negative case comes first)
+
+if (!something || !somethingElse ) {
+    /* do stuff */
+} else {
+    /* do other stuff */
+}
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+### Multiple `if-else`
+
+Creating a chain of conditional statements can make refactoring hard because the decisions are mixed with the actions. Instead, separate out the decisions into functions that can return simple actions:
+
+```js
```

Comment:
I think it'd be helpful to show what `getSomethingAction` looks like to show what the transformation of the "bad" code looks like in the "good" code